src/main/java/frc/robot/commandgroups/bits/CheckArmPositions.java

```diff
@@ -15,6 +14,6 @@ public CheckArmPositions() {
         new WaitCommand(1);
         new SetArmsPosition(new Translation2d(0, 0));
         new WaitCommand(1);
-        new SetArmsPosition(ArmConstants.ARM_DEFAULT_POSITION);
+        new SetArmsPosition(ArmConstants.ABOVE_GAME_PIECE);
```

Comment:
A bit ambiguous -- game piece where? On the floor? On the Substation? Maybe just name this constant "above floor" or something similar.

Reply:
this is the position that's above the game piece in the robot,
where the game pieces from the intake end up. I'll try to think of a less confusing name